packages/tlshelpers/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=tlshelpers
+pkgver=21.6e422be
+pkgrel=1
+pkgdesc='A collection of shell scripts that help handling X.509 certificate and TLS issues.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/hannob/tlshelpers'
+license=('custom:UNLICENSE')
+depends=('der-ascii')
+makedepends=('git')
+source=("git+https://github.com/hannob/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf LICENSE *.md .gitignore
```

Comment:
not useful if not doing a `cp *` afterward

Reply:
> not useful if not doing a `cp *` afterward

Right, I deleted `rm -rf LICENSE *.md .gitignore`. Should I delete also `install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"`?

Reply:
Yes as it's a common license.

Reply:
> Yes as it's a common license.

Deleted as requested.